rundeckapp/grails-spa/packages/ui-trellis/src/app/pages/job/browse/tests/JobScmStatus.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { mount, VueWrapper } from "@vue/test-utils";
+import JobScmStatus from "../tree/JobScmStatus.vue";
+
+jest.mock("@/library/rundeckService.ts", () => ({
+  getRundeckContext: jest.fn().mockImplementation(() => ({
+    eventBus: { on: jest.fn(), emit: jest.fn() },
+    rdBase: "http://localhost:4440",
+  })),
+}));
+const mountJobScmStatus = async (props: any): Promise<VueWrapper<any>> => {
+  const wrapper = mount(JobScmStatus, {
+    props,
+    global: {
+      mocks: {
+        $t: jest.fn().mockImplementation((msg) => msg),
+        $tc: jest.fn().mockImplementation((msg) => msg),
+      },
+    },
+  });
+
+  // Wait for the next Vue tick to allow for asynchronous rendering
+  await wrapper.vm.$nextTick();
+
+  return wrapper;
+};
+
+describe("JobScmStatus", () => {
+  it("no content if there is no scm status", async () => {
+    const wrapper = await mountJobScmStatus({ itemData: {} });
+
+    let detail = wrapper.findAll("*");
```

Comment:
question: why you're searching for an asterisk here?

Reply:
css selector for any element, expecting to find nothing in this case